src/map_server/include/map_server/map_reps/gridmap.h

```diff
@@ -0,0 +1,47 @@
+#ifndef MAP_SERVER_MAP_REPS_GRIDMAP_LOADER_H
+#define MAP_SERVER_MAP_REPS_GRIDMAP_LOADER_H
+
+
+
+// TODO: Port grid_map to ROS2 //
+
+#include "map_server/base_map_loader.h"
+#include <grid_map_msgs/GridMap.h>
+#include <grid_map_ros/grid_map_ros.hpp>
+#include <grid_map_cv/grid_map_cv.hpp>
+#include <cv_bridge/cv_bridge.h>
```

Comment:
There seem to be several headers included that aren't required by the definition of this interface. Only include header files required by the interface. The implementation file can include additional headers, if required by the implementation.